### DISCUSSION TITLE
Multiline format based on exit code

### DISCUSSION THREAD

**Discussion by surgicalmittens** [starship/starship#6360] [Q&A]
Hi, 

I'm hoping someone can help a starship newb with a question. 

I've got a multi-line starship format line. The second line changes to red or green based on the character success_symbol/exit_symbol.

```
[character]
success_symbol = '[╰⎯](bold green) '
error_symbol = '[╰⎯](bold red) '
```

I've also made a status block that will change colored output based on the exit code of the prior command. 

```
[status]
#style = 'bg:blue'
symbol = '[╭╴](bold red)'
success_symbol = '[╭╴](bold green)'
```

My format string for the prompt looks as follows. 
```
format = """$status$os$username$hostname$kubernetes$directory$git_branch$git_status$python
$character
"""
```

This prints almost everything correctly, other than including the status code itself in the prompt. I was wondering if there is a way to either remove the status code or a better way to colorize the first character. 

Essentially, I would like the 127 and 0 gone from the output. 

![image](https://github.com/user-attachments/assets/697412ea-9b55-427a-80f1-b8fbed2062c1)

Thanks for your time and assistance.

**Comment by d0mb1** [comment]
I've been wondering the same thing and found a solution.

```
[status]
format = "[$symbol ]($style)"
symbol = '[your prompt](your colour)'
success_symbol = '[your prompt](your colour)
disabled = false
```
You have to specify in the format not to add the error code.
Hope this helps!
Also if anyone knows how to change the color of `[fill]` based on the color of `[character]`/`[status]` that would be much appreciated <3
